<launch>
    <node pkg="rmcore" type="derivative_test" name="derivative_test" required="true">
        <rosparam command="load" file="$(find rmcore)/cfg/config.yaml"/>
    </node>
</launch>
